modules/pensions/lib/pensions/pdf_fill/va21p527ez.rb

```diff
@@ -9,6 +9,7 @@
 require_relative 'helpers'
 
 # rubocop:disable Metrics/ClassLength
+# rubocop:disable Metrics/MethodLength
```

Comment:
I am planning on breaking this file up into 12 smaller files (one for each section) and most of the expand functions will be getting broken up as well so this is only here TEMPORARILY to keep the scope of the PR small